onecodex/cli.py

```diff
@@ -492,6 +498,40 @@ def upload(
     )
 
 
+@onecodex.command("upload_asset")
```

Comment:
For consistency, there should probably be an `asset` command group with `upload` command put inside, e.g.
`onecodex assets upload` vs `onecodex upload_asset`

Reply:
Good point - it's already a little inconsistent with `upload` (for FASTA/FASTQ) and then `documents upload`, but I agree having `assets upload` would be better. Will update.